src/convnets/unet.jl

```diff
@@ -0,0 +1,73 @@
+function unet_block(in_chs::Int, out_chs::Int, kernel = (3, 3))
+	return Chain(conv1 = Conv(kernel, in_chs => out_chs; pad = (1, 1)),
+		norm1 = BatchNorm(out_chs, relu),
+		conv2 = Conv(kernel, out_chs => out_chs; pad = (1, 1)),
+		norm2 = BatchNorm(out_chs, relu))
```

Comment:
Any specific reason for using named layers? I don't see it being used anywhere. I haven't seen Metalhead use such a code convention, so a seems a little inconsistent with the code base.

Reply:
Agree with this, it looks weird – I think the only place we might need named layers is if we specifically need to index into a `Chain` later for use and the name of the layer isn't apparent. Here I don't see that happening.